solutions/ac-matching.mdx

```diff
@@ -35,115 +36,84 @@ considered as a single matching involving zero pairs).
 #include <bits/stdc++.h>
 using namespace std;
 
-bool compat[21][21];
-int dp[1 << 21];
+const int MOD = 1e9 + 7;
+const int MAX_N = 21;
 
-int N;
+bool compat[MAX_N][MAX_N];
+int dp[1 << MAX_N];
 
 int main() {
-	cin.tie(0)->sync_with_stdio(0);
```

Comment:
why? D:

Reply:
isn't necessary

Reply:
so? might as well include it as part of every C++ program :c

(ah well idrc, if I'm the only one who feels this way then whatever)

Reply:
doesnt it speed io up?

Reply:
yes, but n is like <= 20

> doesnt it speed io up?

Reply:
oh yeaa xd. ig it doesn't rly matter if we put this, but it does look cool 😆

Reply:
yeah same I am quite impartial on this :laughing: